src/App.css

```diff
@@ -1,39 +1,150 @@
+@import url('https://fonts.googleapis.com/css2?family=Roboto+Slab:wght@100;200;300;400;500;600;700;800;900&display=swap');
```

Comment:
- [ ] Kindly use the correct styles `font family: Lato and gill sans` to be similar to the original design 👇🏽 

![image](https://user-images.githubusercontent.com/58642949/217983141-77396687-1df7-44b7-bfb1-e79bf6315be9.png)

> You should follow these [design guidelines](https://www.behance.net/gallery/31579789/Ballhead-App-(Free-PSDs)), including:
> Colors (select one main color for your website).
> Typography: font face, size and weight.
> Layout: composition and space between elements.